OneSignalSDK/unittest/src/test/java/com/test/onesignal/MainOneSignalClassRunner.java

```diff
@@ -877,6 +877,17 @@ public void testOnSessionCalledOnlyOncePer30Sec() throws Exception {
       assertEquals("{\"app_id\":\"b4f7f966-d8cc-11e4-bed1-df8f05be55ba\"}", ShadowOneSignalRestClient.lastPost.toString());
    }
 
+   @Test
+   public void testRequestMadeWithCorrectTimeZoneID() throws Exception {
+      // Will call create
+      TimeZone.setDefault(TimeZone.getTimeZone("Europe/London"));
+      OneSignalInit();
+      threadAndTaskWait();
+      ShadowOneSignalRestClient.lastUrl = null;
+      JSONObject jsonObject = new JSONObject(ShadowOneSignalRestClient.lastPost.toString());
```

Comment:
no need for a new JSONObject, ShadowOneSignalRestClient.lastPost is already a JSONObject

Reply:
Did you check this comment?